pkg/cdclog/puller.go

```diff
@@ -61,10 +61,12 @@ func NewEventPuller(
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
-		ddlFileIndex++
```

Comment:
what if the len(ddlFiles[0]) == 0 and len(ddlFiles[1]) > 0. Can this happened?

Reply:
This shouldn't happen because we sort the ddl files by its' event timestamp.